static/src/stylesheets/layout/nav/_veggie-burger.scss

```diff
@@ -12,8 +12,9 @@
     z-index: $zindex-main-menu - 1;
 
     // menu is open
-    .new-header--open & {
-        z-index: $zindex-main-menu + 1;
+    .new-header--open &,
```

Comment:
So glad the new-header isn't the new header

Reply:
I almost went with `newest-header` but then thought of future travellers ;)